Gemfile

```diff
@@ -1,8 +1,16 @@
 source 'https://rubygems.org'
 
+gem 'pg'
```

Comment:
I'm having trouble loading it via ruby app.rb. Maybe specify a version in the gemfile?